Doug/Items/Consumables/VapeMachine.cs

```diff
@@ -0,0 +1,36 @@
+using Doug.Effects.Buffs;
+using Doug.Models;
+using Doug.Repositories;
+
+namespace Doug.Items.Consumables
+{
+    public class VapeMachine : ConsumableItem
+    {
+        public const string ItemId = "vape_machine";
+
+        private readonly IEffectRepository _effectRepository;
+        private const int DurationInMinutes = 5;
+
+        public VapeMachine (IInventoryRepository inventoryRepository, IEffectRepository effectRepository) : base(inventoryRepository)
+        {
+            _effectRepository = effectRepository;
+            Id = ItemId;
+            Name = "Vape Machine";
+            Description = "Some kind of hobo breathing mechanism";
+            Rarity = Rarity.Uncommon;
+            Icon = ":smoking:";
+            Price = 15;
```

Comment:
This item is worth a bit more than 15 I think!

Reply:
I just upgrade to 40